lib/course_planner/Mailer/main.ex

```diff
@@ -0,0 +1,6 @@
+defmodule CoursePlanner.Mailer.Main do
```

Comment:
I named it based on folder structure. if I change Main and Mailer then it does not represent the structure. do you know a better name instead of Main ?